applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -333,40 +333,6 @@ class MechanicalExplicitStrategy
         KRATOS_CATCH("")
     }
 
-    /**
-     * @brief This method add the contributions of the residual
-     * @param pScheme The integration scheme considered
-     * @param rModelPart The model of the problem to solve
-     */
-    void CalculateAndAddRHS(
```

Comment:
why is this removed?

Reply:
this is now in the scheme. I need to call it multiple times in the new scheme.